src/InflectorConfig.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace TomPHP\ConfigServiceProvider;
+
+use ArrayIterator;
+use IteratorAggregate;
+
+final class InflectorConfig implements IteratorAggregate
+{
+    /** @var array */
+    private $inflectors;
+
+    /**
+     * @param array $config
+     */
```

Comment:
@tomphp 

Since 

* all of the arguments are sufficiently type-hinted
* the method doesn't return anything

can we drop the docblock?